Features/Context/SubContext/CommonActions.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+/**
+ * File containing the Common Functions for context class PlatformUI.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\PlatformUIBundle\Features\Context\SubContext;
+
+trait CommonActions
+{
+    /**
+     * @Given I click (on) the logo
+     * Clicks on the PlatformUI logo
+     */
+    public function clickLogo()
+    {
+        $page = $this->getSession()->getPage();
+        $selector = '.ez-logo a';
+        $page->find('css', $selector)->click();
+    }
+
+    /**
+     * @Given I click (on) the tab :tab
+     * Clicks on a PlatformUI tab
+     *
+     * @param   string  $tab    Text of the element to click
+     */
+    public function clickTab($tab)
+    {
+        $this->clickElementByText($tab, ".ez-tabs-label a[href]");
+    }
+
+    /**
+     * @Given I click (on) the navigation zone :zone
+     * Click on a PlatformUI menu zone
+     *
+     * @param   string  $zone   Text of the element to click
+     */
+    public function clickNavigationZone($zone)
+    {
+        $this->clickElementByText($zone, ".ez-zone-name");
+    }
+
+    /**
+     * @Given I click on the :button button number :index
+     * Click on a PlatformUI button
+     *
+     * @param  string   $button     Text of the element to click
+     * @param  string   $index      WHAT IS THIS?!
+     */
+    public function clickButtonWithIndex($button, $index)
+    {
+        $this->clickElementByText($button, "button", $index);
+    }
+
+    /**
+     * @Given I over (on) the navigation zone :zone
+     * Over on a PlatformUI menu zone
+     *
+     * @param  string   $button     Text of the element to click
+     */
+    public function overNavigationZone($zone)
```

Comment:
This is a piece of sneaky code that survived the clean up :sweat_smile:. Removed!